src/timepicker/docs/readme.md

```diff
@@ -0,0 +1,12 @@
+In the time entry field allow user to enter the time in several formats. When user have entered the value, change formatting to the 24:00 clock. 
```

Comment:
I can see this is copied directly from the user story. Wording should be made for public use.